src/main/java/edu/tamu/scholars/middleware/export/utility/FilenameUtility.java

```diff
@@ -0,0 +1,43 @@
+package edu.tamu.scholars.middleware.export.utility;
+
+import java.util.Date;
+import java.util.Locale;
+
+import org.springframework.context.i18n.LocaleContextHolder;
+
+import edu.tamu.scholars.middleware.discovery.model.AbstractIndexDocument;
+import edu.tamu.scholars.middleware.discovery.model.Individual;
+import edu.tamu.scholars.middleware.discovery.model.helper.IndividualHelper;
+
+public class FilenameUtility {
+
+    private final static String SPACE = " ";

```

Comment:
Already exists in Apache lang dependency [StringUtils](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html). Some other useful methods for cleaning up code as well.